src/Cli/dotnet/ReleasePropertyProjectLocator.cs

```diff
@@ -145,10 +147,11 @@ public IEnumerable<string> GetCustomDefaultConfigurationValueIfSpecified()
         /// Throws exception if two+ projects disagree in PublishRelease, PackRelease, or whatever _propertyToCheck is, and have it defined.</returns>
         public ProjectInstance? GetArbitraryProjectFromSolution(string slnPath, Dictionary<string, string> globalProps)
         {
-            SlnFile sln;
+            string slnFullPath = Path.GetFullPath(slnPath);
```

Comment:
Is slnPath guaranteed to be real here? I'm curious if we should have some kind of a "ensure we can really find file" check